source/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,199 @@
+.. _tls:
+
+==============================
+Enable TLS/SSL on a Connection
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can connect to MongoDB instances with the
+:wiki:`TLS/SSL </Transport_Layer_Security>` security protocol using
+the underlying TLS/SSL support in the JDK. To configure your connection
+to use TLS/SSL, enable the TLS/SSL settings in either the
+:java-sync-api:`ConnectionString
+<com/mongodb/ConnectionString.html>` or :java-sync-api:`MongoClientSettings
+<com/mongodb/MongoClientSettings.html>` objects.
+
+.. note:: Debugging TLS/SSL
+
+   If you experience trouble setting up your TLS/SSL connection, you can
+   use the ``-Djavax.net.debug=all`` system property to view additional
+   log statements. See `the Oracle guide to debugging TLS/SSL connections
+   <https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ReadDebug.html>`__
+   for more information.
+
+.. _tls-enable:
+
+Enable TLS/SSL
+--------------
+
+You can enable TLS/SSL for the connection to your MongoDB instance
+in two different ways: through a parameter in your connection string, or
+using a method in the ``MongoClientSettings.Builder`` class.
+
+.. tabs::
+
+   .. tab:: ConnectionString
+      :tabid: connectionstring
+
+      To enable TLS/SSL on a connection with a :java-sync-api:`ConnectionString
+      <com/mongodb/ConnectionString.html>`, assign the connection string
+      parameter ``tls`` a value of ``true`` in the connection string passed to
+      ``MongoClients.create()``:
+
+      .. code-block:: java
+
+         MongoClient mongoClient = MongoClients.create("mongodb+srv://<user>:<password>@<cluster-url>?tls=true");
+
+   .. tab:: MongoClientSettings
+      :tabid: mongoclientsettings
+
+      To configure your ``MongoClient``'s TLS/SSL connection options using the
+      ``MongoClientSettings.Builder`` class, call the :java-sync-api:`applyToSslSettings()
+      <com/mongodb/MongoClientSettings.Builder.html#applyToSslSettings(com.mongodb.Block)>`
+      method. Set the ``enabled`` property to ``true`` in the ``SslSettings.Builder``
+      block to enable TLS/SSL:
+
+      .. code-block:: java
+
+         MongoClientSettings settings = MongoClientSettings.builder()
+                .applyToSslSettings(builder -> 
+                    builder.enabled(true))
+                .build();
+         MongoClient client = MongoClients.create(settings);
+
+.. _tls_configure-certificates:
+
+Configure Certificates
+----------------------
+
+Java applications that serve TLS/SSL requests require access to
+cryptographic certificates that prove identity for the application
+itself as well as other applications with which the application
+interacts. To configure access to these certificates in your application,
+you should use the JVM Trust Store and the JVM Key Store.
+
+.. _tls-configure-jvm-truststore:
+
+Configure the JVM Trust Store
```

Comment:
We should note that by default the JRE includes most commonly used public certificates from signing authorities, e.g. Let's Encrypt.  So by default you don't have to do anything here.  e.g. to connect to an Atlas server it's enough to just enable TLS and it will just work.